pkg/microservice/aslan/core/delivery/handler/router.go

```diff
@@ -45,6 +45,14 @@ func (*Router) Inject(router *gin.RouterGroup) {
 		deliveryRelease.GET("/:id", GetDeliveryVersion)
 		deliveryRelease.GET("", ListDeliveryVersion)
 		deliveryRelease.DELETE("/:id", GetProductNameByDelivery, gin2.UpdateOperationLogStatus, DeleteDeliveryVersion)
+
+		deliveryRelease.POST("/helm", CreateHelmDeliveryVersion)
```

Comment:
/helm/versions

Reply:
api url paths will be adjusted in next feature develop pr